entity-framework/core/providers/sql-server/temporal-tables.md

```diff
@@ -0,0 +1,282 @@
+﻿---
+title: Microsoft SQL Server Database Provider - Temporal Tables - EF Core
+description: Using SQL Server/Azure SQL temporal tables (aka system-versioned tables) with EF Core
+author: ajcvickers
+ms.date: 02/11/2023
+uid: core/providers/sql-server/temporal-tables
+---
+
+# SQL Server/Azure SQL temporal tables
+
+[SQL Server temporal tables](/sql/relational-databases/tables/temporal-tables) automatically keep track of all data ever stored in a table, even after that data has been updated or deleted. This is achieved by creating a parallel "history table" into which timestamped historical data is stored whenever a change is made to the main table. This allows historical data to be queried, such as for auditing, or restored, such as for recovery after accidental mutation or deletion.
+
+EF Core supports:
+
+* The creation of temporal tables using [EF Core Migrations](xref:core/managing-schemas/migrations/index)
+* Transformation of existing tables into temporal tables, again using Migrations
+* Querying historical data
+* Restoring data from some point in the past
```

Comment:
Is "reading" more accurate than "restoring"? Restoring sounds like the ability to revert the entire table to some past state.

Reply:
Restoring is what I show in the example. That is, reading data from some previous point in time and then updating the current state with that data.